addons/account/static/src/components/tax_totals/tax_totals.js

```diff
@@ -17,8 +16,8 @@ class TaxGroupComponent extends Component {
         this.state = useState({ value: "readonly" });
         onPatched(() => {
             if (this.state.value === "edit") {
-                const { taxGroup, currency } = this.props;
-                const newVal = formatFloat(taxGroup.tax_group_amount, { digits: currency.digits });
```

Comment:
I think you shouldn't remove the `digits` part because if you do, it will always be rounded to 2 instead of the currency precision.

Reply:
Actually, this PR was originally in #105469, which contains a better explanation.
I thought that mentioning it would be enough, but I guess I was wrong.
Thanks for the advice 😄.

As for the currency prop in `TaxGroupComponent`, I talked to @h4818 about this, and if I remember correctly, he told me that it wasn't really needed there.

Reply:
We need to checkout, If the formatting is perfect in: 
-both edit & readonly mode
-with a currency and without
-with a currency with more / less than 2 digits

then it should be ok to remove it. otherwise keep the digits option.

Reply:
do we need a test?

Reply:
I wouldn't add a test for removing a reference to a dead variable.
But, for safety, I would keep the digits part if we don't have some. (But I haven't that many knowledge in that area of the code)